src/Components/Hero/Hero.test.tsx

```diff
@@ -0,0 +1,9 @@
+﻿import {render} from "@testing-library/react";
+import {Curiosity} from "../../Pages/CuriosityLandingPage/CuriosityLandingPage";
+import React from "react";
+
+test('check Hero component renders on page', () => {
+    let curiosityPage = render(<Curiosity />);
+    const hero = curiosityPage.getByTestId("hero");
+    expect(hero).toBeInTheDocument();
+});
```

Comment:
Looks like a good test, but I think its really testing the Curiosity component rather than the hero component.
Could you move it to the tests for that page?

Reply:
All done, please review :)